src/logging.rs

```diff
@@ -45,7 +45,10 @@ macro_rules! enable_logging_with_filter {
     ($filter:expr) => {
         pretty_env_logger::formatted_builder()
             .write_style(pretty_env_logger::env_logger::WriteStyle::Auto)
-            .filter(Some(env!("CARGO_PKG_NAME")), $filter)
+            .filter(
+                Some(&env!("CARGO_PKG_NAME").replace("-", "_")),
+                log::LevelFilter::Info,
```

Comment:
Just ran into this bug and found this issue. Just wanted to note that this fix ignores the passed `$filter` and uses `LevelFilter::Info`. This doesn't seem intentional.

Reply:
That's definitely unwanted. I'll shoot a PR.

Reply:
Yes, I've already fixed it: https://github.com/teloxide/teloxide/commit/c5ef0a6c546a75bdb16b0a087b2532e41e506d83#diff-b69620054bd61a3dc57a9b26b88350e5.